api/Window.json

```diff
@@ -5709,6 +5709,131 @@
           }
         }
       },
+      "URL": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/URL",
+          "support": {
+            "chrome": {
+              "version_added": "41",
```

Comment:
I think this got collapsed into one entry erroneously. I think the final result should be:
```
"chrome": [
  {
    "version_added": "41"
  },
  {
    "version_added": "8",
    "prefix": "webkit"
  }
]
```